xbmc/GUIInfoManager.cpp

```diff
@@ -7174,7 +7174,7 @@ bool CGUIInfoManager::GetBool(int condition1, int contextWindow, const CGUIListI
       bReturn = g_application.m_pPlayer->GetPlaySpeed() < 0;
       break;
     case PLAYER_FORWARDING:
-      bReturn = g_application.m_pPlayer->GetPlaySpeed() > 1.5;
+      bReturn = g_application.m_pPlayer->GetPlaySpeed() > IPlayer::MaxPlaySpeed();
```

Comment:
I know setting this to `1.55` would changed behavior, but I don't understand why rounding errors must not be considered here.